main.py

```diff
@@ -1,10 +1,37 @@
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
+import json
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        read_file = json.load(file)
+        for nickname, player_info in read_file.items():
```

Comment:
do not overwhelm the context manager.
first, read the data and then do manipulation with it

Reply:
but I read with help "load", isn't it?